testsuite/integration/manualmode/src/test/java/org/jboss/as/test/manualmode/weld/builtinBeans/LegacyCompliantPrincipalPropagationTestCase.java

```diff
@@ -80,16 +78,11 @@ public static Archive<?> createTestArchive() {
     @Test
     @InSequence(1)
     @RunAsClient
-    public void startContainerAndConfigureApplicationSecDomain() throws Exception {
+    public void startContainer() throws Exception {
         if (!serverController.isStarted(DEFAULT_FULL_JBOSSAS)) {
             serverController.start(DEFAULT_FULL_JBOSSAS);
         }
-        // add application-security-domain called "other" that is mapped with an Elytron security domain (ApplicationDomain) in the Jakarta Enterprise Beans subsystem
         ManagementClient managementClient = getManagementClient();
-        ModelNode modelNode = createAddOperation(PathAddress.parseCLIStyleAddress(("/subsystem=ejb3/application-security-domain=other")).toModelNode());
-        modelNode.get("security-domain").set("ApplicationDomain");
-        managementClient.getControllerClient().execute(modelNode);
-
         ServerReload.reloadIfRequired(managementClient);
```

Comment:
This reload can be removed too since we're no longer adding the `application-security-domain` resource.